examples/alias/versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 5.70"
+      version = ">= 5.79"
```

Comment:
This is a breaking change.
This changes should be noted in CHANGELOG.md.
https://github.com/terraform-aws-modules/terraform-aws-lambda/blob/master/CHANGELOG.md#7170-2024-12-08